mungegithub/mungers/close-stale-obj.go

```diff
@@ -30,83 +30,55 @@ import (
 )
 
 const (
-	day              = time.Hour * 24
-	keepOpenLabel    = "keep-open"
-	stalePullRequest = 90 * day // Close the PR if no human interaction for `stalePullRequest`
-	startWarning     = 60 * day
-	remindWarning    = 30 * day
-	closingComment   = `This PR hasn't been active in %s. Closing this PR. Please reopen if you would like to work towards merging this change, if/when the PR is ready for the next round of review.
+	day            = time.Hour * 24
+	keepOpenLabel  = "keep-open"
+	stalePeriod    = 90 * day // Close the PR/Issue if no human interaction for `stalePeriod`
+	startWarning   = 60 * day
+	remindWarning  = 30 * day
+	closingComment = `This PR/Issue hasn't been active in %s. Closing this PR/Issue. Please reopen if you would like to work towards merging this change, if/when the PR/Issue is ready for the next round of review.
```

Comment:
Yes, use "Issue" or "PR" as appropriate.

Oh, they're already put in Printf, so you could turn it into `This %s hasn't been...`